AnimView/frmSprites.h

```diff
@@ -23,6 +23,7 @@ SOFTWARE.
 #pragma once
 
 #include "config.h"
+
```

Comment:
I wonder about the `#pragma once` line, which is non-standard cpp afaik. Wikipedia claims it to be an alternative to the `#ifndef/#define/#endif` lines, which I just merged a fix for.
According to https://en.wikipedia.org/wiki/Pragma_once#Portability portability looks like a non-issue, but perhaps settle on one form?

Reply:
Right, should replace that pragma. Thanks.